src/modules/drive/Toolbar/components/CreateExcalidrawItem.spec.jsx

```
@@ -0,0 +1,63 @@
+}))
+
+describe('CreateExcalidrawItem', () => {
+  beforeEach(() => {
+    jest.clearAllMocks()
+    mockParams = { folderId: 'folder123' }
+  })
+
+  it('should navigate to the create route for the current folder', () => {
+    const { getByTestId } = render(
+      <CreateExcalidrawItem onClick={jest.fn()} />
+    )
```

ESLint

[error] Replace `⏎······<CreateExcalidrawItem·onClick={jest.fn()}·/>⏎····` with `<CreateExcalidrawItem·onClick={jest.fn()}·/>`

(prettier/prettier)